src/utils/sykmeldinger/sykmeldingUtils.ts

```diff
@@ -147,6 +147,11 @@ export const arbeidsgivernavnEllerArbeidssituasjon = (
     sykmelding.innsendtArbeidsgivernavn.length > 0
   ) {
     return sykmelding.innsendtArbeidsgivernavn;
+  } else if (
+    sykmelding.status === SykmeldingStatus.NY &&
+    sykmelding.arbeidsgiver
+  ) {
+    return `${sykmelding.arbeidsgiver} (fylt inn av sykmelder)`;
```

Comment:
Snakket litt med team sykemdling, Anders og Stine og gjorde noen endringer.
Hvis status = `NY`/`APEN`, så er den ikke tatt i bruk, men vi har heller ikke noe info om arbeidsgiver. Det eneste vi vet er hva legen har fylt inn i sykmeldingen som arbeidsgiver (kan være feil). Når sykmeldingen er tatt i bruk, vil vi få riktig arbeidsgiver og statusen for sykmeldingen vil være `SENDT`.

<img width="1242" alt="image" src="https://github.com/navikt/syfomodiaperson/assets/37441744/adc06167-13b1-4215-bfcf-258aadb09db2">